scripts/configure_petsc.sh

```diff
@@ -44,7 +50,7 @@ function configure_petsc()
       --with-fortran-bindings=0 \
       --with-sowing=0 \
       --with-64-bit-indices \
-      $*
+      "$@"
```

Comment:
Back to this?

Reply:
Yeah, we keep running into needing it, so it probably needs to remain....at least the quotes. We might be able to get by with `